### DISCUSSION TITLE
Operator scope

### DISCUSSION THREAD

**Discussion by ennc0d3** [emqx/emqx-operator#1044] [Q&A]
I am using the emqx-operator (2.2.14) installed using Helm, which by default its watches all namespaces. Now I want  to use 2.2.20 version and deploy EMQX cluster to a different namespace to isolate and test. How can i do that?

Ref:  https://github.com/emqx/emqx-operator/issues/452, the ticket says is not possible to have two operator instances, but as this old am asking again. If we can do something with tuning the ServiceAccount/RBAC?

**Comment by Rory-Z** [comment]
Sorry, one k8s cluster only can deploy one EMQX operator, because EMQX operator need EMQX CRD, and EMQX CRD is a cluster resource, it cannot assign some namespace.
